web/migrations/0050_pdftype_pdfsubmission.py

```
@@ -0,0 +1,102 @@
+                (
+                    "student",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        related_name="pdf_submissions",
+                        to=settings.AUTH_USER_MODEL,
+                    ),
+                ),
+                (
+                    "pdf_type",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.PROTECT, related_name="submissions", to="web.pdftype"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)